doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -38,6 +38,10 @@ to the maintenance branch, and later will be forward ported to master.
 Finish the Release Note
 -----------------------
 
+.. note:
+
+  This has changed now that we use ``towncrier``
```

Comment:
Might be a bit cryptic without information on where to look?

Reply:
adding the path to the `README.rst`